core/src/test/java/com/yahoo/imapnio/async/exception/ImapAsyncClientExceptionTest.java

```diff
@@ -43,7 +43,7 @@ public void testImapAsyncClientExceptionWithFailureTypeAndCause() {
     @Test
     public void testImapAsyncClientExceptionWithSessionIdClientContext() {
         final ImapAsyncClientException.FailureType failureType = ImapAsyncClientException.FailureType.CHANNEL_DISCONNECTED;
-        final Long sessionId = new Long(5);
+        final Long sessionId = 5L;
```

Comment:
revert

Reply:
Avoid autoboxing or unboxing in code, even in UT.  autoboxing and unboxing is at run time.  Lets avoid.